tests/cloud_playbooks/roles/packet-ci/templates/inventory.j2

```diff
@@ -125,11 +83,6 @@ instance-1 etcd_member_name=etcd2
 instance-2 etcd_member_name=etcd3
 {% endif %}
 
-[k8s-cluster:children]
-kube-node
-kube-master
-calico-rr
-
 [k8s_cluster:children]
 kube_node
 kube_master
```

Comment:
Shouldn't this be `kube_control_plane` ? (cf CI error as this
```
- name: Create kubernetes directories
  file:
    path: "{{ item }}"
    state: directory
    owner: kube
  when: inventory_hostname in groups['k8s_cluster']
```
Doesn't loop on control planes item, only on node

Reply:
ah, nice catch!
Let me try it.